### PR TITLE
Add PCA tracker

### PR DESCRIPTION
**Pull request by ProfFan**

### CODE REVIEW

File: Examples/OISTVisualizationTool/Util.swift [hunk 1]
```diff
@@ -95,6 +95,58 @@ public func makeNaiveBayesRAETracker(
     })
 }
 
+/// Returns a tracking configuration for a tracker using PCA.
+///
+/// Parameter model: The PCA model to use.
+/// Parameter statistics: Normalization statistics for the frames.
+/// Parameter frames: The frames of the video where we want to run tracking.
+/// Parameter targetSize: The size of the target in the frames.
```
Comment (dellaert) [inline Examples/OISTVisualizationTool/Util.swift:103]: copy/pasted from the wrong thing: fg and bg models missing

File: Examples/OISTVisualizationTool/main.swift [hunk 2]
```diff
@@ -510,6 +506,153 @@ struct TrainRAE: ParsableCommand {
   }
 }
 
+/// PPCA with probabilistic model
+///
+/// Tracking with a Naive Bayes with RAE
+struct NaivePca: ParsableCommand {
```
Comment (dellaert) [inline Examples/OISTVisualizationTool/main.swift:512]: Hmm :-) I feel there is a lot of copy/paste here again, so I guess my previous request to split up in multiple functions might not have made it? By copy/pasting that mess we have created more work for us in the future.

**Review by dellaert — COMMENTED** [review]
A bit grumpy but will relent to deadline pressure :-)

File: Examples/OISTVisualizationTool/main.swift (hunk 2 above)
Comment (ProfFan) [reply Examples/OISTVisualizationTool/main.swift:512]: really don't have time, it's due to the inability to make type erased container for probability distributions

Comment (ProfFan) [reply Examples/OISTVisualizationTool/main.swift:512]: * in short time

**Pull request merged**